src/framework/midi/internal/platform/win/winmidioutport.cpp

```diff
@@ -149,6 +149,11 @@ MidiDeviceID WinMidiOutPort::deviceID() const
     return m_deviceID;
 }
 
+bool WinMidiOutPort::supportsMIDI20Output() const
+{
+    return false;
```

Comment:
Doesn't Windows support midi 2.0?

Reply:
At least our implementation of the Windows MIDI port converts everything to MIDI1.0 before sending it. So we don't really support sending MIDI2.0 events. If Windows itself supports MIDI2.0 (I don't know this), we can update our implementation to make use of it, and then we can change this to `true`.